fastlane/swift/SocketClient.swift

```diff
@@ -302,8 +302,9 @@ extension SocketClient: StreamDelegate {
                 self.closeSession(sendAbort: false)
             }
 
-        case let .failure(failureInformation):
-            socketDelegate?.commandExecuted(serverResponse: .serverError) {
+        case let .failure(failureInformation, failureClass, failureMessage):
+          LaneFile.fastfileInstance?.onError(currentLane: ArgumentProcessor(args: CommandLine.arguments).currentLane, errorInfo: failureInformation.joined(), errorClass: failureClass, errorMessage: failureMessage)
+          socketDelegate?.commandExecuted(serverResponse: .serverError) {
```

Comment:
Looks like the indentation here is wrong (for Swift)? 🙈 👀

Reply:
🤔 maybe, anyway, there's a linter once this is merged so I don't think it is much trouble. In fact I think this is the xcode indentation that is slightly different than Swiftformat's one. Should I change it?